web/src/main/java/org/mskcc/oncotree/OncotreeApplication.java

```diff
@@ -41,7 +41,7 @@ public void addCorsMappings(CorsRegistry registry) {
             @Override
             public void addViewControllers(final ViewControllerRegistry registry) {
                 super.addViewControllers(registry);
-                registry.addRedirectViewController("/oncotree-mappings/","/oncotree-mappings.html").setKeepQueryParams(true).setStatusCode(HttpStatus.MOVED_PERMANENTLY);
+                registry.addRedirectViewController("/oncotree-mappings/","/#/home?tab=mapping").setKeepQueryParams(true).setStatusCode(HttpStatus.MOVED_PERMANENTLY);
```

Comment:
We should also have a sysadmin update the apache config to forward to this new spot from the original URL directory for the prior webapp.